lib/os/thread_posix.cpp

```diff
@@ -30,9 +30,15 @@
 #    include <pthread.h>
 
 #    ifdef __linux__
-#        include <linux/unistd.h>
-#        include <sys/syscall.h>
-#        include <unistd.h>
+#        ifdef PSTORE_HAVE_LINUX_UNISTD_H
```

Comment:
These includes no longer need to be guarded by __linux__ if we’re checking for their presence at configure time.

Reply:
Thanks, fixed in the commit https://github.com/SNSystems/pstore/pull/93/commits/ed5cf97fa57af25655999699db16ec75c2c7fe96.